vm/core/context_test.go

```diff
@@ -3,30 +3,28 @@ package core_test
 import (
 	"testing"
 
-	"github.com/spacemeshos/go-scale"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/mock/gomock"
 
+	"github.com/spacemeshos/go-spacemesh/genvm/core"
+	"github.com/spacemeshos/go-spacemesh/genvm/core/mocks"
```

Comment:
I think it should import the `vm/core`, not the `genvm/core`. It's currently testing the wrong thing :)
```suggestion
	"github.com/spacemeshos/go-spacemesh/vm/core"
	"github.com/spacemeshos/go-spacemesh/vm/core/mocks"
```